docs/schema/2.0/manifest.schema.json

```diff
@@ -69,7 +69,9 @@
               "type": "string"
             }
```

Comment:
I'm wondering if we are versioning the schema somehow?
I see the `$id` might be usable if we picked SemVer. 
E.g. instead of 
```
"$id": "https://github.com/getoutreach/extensibility-sdk/docs/schema/2.0/manifest.schema.json",
```
...we'd use `2.0.1`
```
"$id": "https://github.com/getoutreach/extensibility-sdk/docs/schema/2.0.1/manifest.schema.json",
```

Or did we decide not to create a new version unless a breaking change is introduced?

Reply:
TBH I seriously doubt that we use it somewhere and that it is up-to-date.